api/fees/data.go

```diff
@@ -148,22 +148,27 @@ func (fd *FeesData) getRewardsForCache(block *block.Block) (*rewards, error) {
 	}
 	state := fd.stater.NewState(parentSummary.Root())
 
-	baseGasPrice, err := builtin.Params.Native(state).Get(thor.KeyBaseGasPrice)
+	legacyTxDefaultGasPrice, err := builtin.Params.Native(state).Get(thor.KeyLegacyTxDefaultGasPrice)
```

Comment:
```suggestion
	legacyTxBasetGasPrice, err := builtin.Params.Native(state).Get(thor.KeyBaseGasPrice)
```

I would suggest rename the variable to `legacyTxBasetGasPrice` as the distinction to `block basefee` and keep the parameter name as is.

Reply:
I have a concern because **Base** is used in 2 contexts, of the block **base** fee and of the legacy transaction **base** fee.

I believe I've spotted some places where they were switched, so my preference is to keep block base and to change the legacy from base to default. But I'd be happy to call it something.

Reply:
Updated name to LegacyTxBaseGasPrice 👍